packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -133,6 +135,35 @@ const TradingAppCard = ({
     const is_disabled = !!(mt5_acc_auth_status && !migration_status) && !is_eu_user;
     const platform_name = is_account_being_created ? name : sub_title ?? name;
 
+    const renderStatusBadge = () => {
+        if (mt5_acc_auth_status) {
+            return (
+                <StatusBadge
+                    className='trading-app-card__acc_status_badge'
+                    account_status={mt5_acc_auth_status}
+                    icon={badge_icon}
+                    text={badge_text}
+                    onClick={() => handleStatusBadgeClick(mt5_acc_auth_status)}
+                />
+            );
+        }
+
+        if (server_disabled) {
+            const text = 'Disabled';
```

Comment:
Put the disabled directly in the text, making another variable does not make sense